workload-identity/bind-service-accounts.sh

```diff
@@ -98,12 +98,14 @@ else
   sleep 2m
 fi
 
+
 pod-identity() {
   head -n 1 <(
     entropy=$(date +%S)
     set -o xtrace
     kubectl run --rm=true -i \
-      "--context=$context" "--namespace=$namespace" "--serviceaccount=$name" \
+      "--context=$context" "--namespace=$namespace" \
+      --overrides="{\"spec\": {\"serviceAccount\": \"$name\"}}" \
```

Comment:
Nit: I would change `--overrides="` to `"--overrides=`.

Reply:
Done.